e2e/tests/ui_tests/smoke_test.js

```diff
@@ -55,6 +55,6 @@ Scenario('GA for 1v1- respond to application - Request more information', async
   console.log('Responded to Judge Additional Information on case: ' + childCaseNum());
 }).retry(0);
 
-AfterSuite(async ({api}) => {
-  await api.cleanUp();
-});
+// AfterSuite(async ({api}) => {
```

Comment:
@Azam-Hmcts  is this intentional?

Reply:
Yeah, it was. But good eye. Its not more needed.